README.md

```diff
@@ -156,6 +156,17 @@ forge script script/DeployAllContracts.s.sol:DeployAllContracts --rpc-url $ETH_R
 
 For Etherscan verification, ensure that the `ETHERSCAN_API_KEY` environment variable is set and add the `--verify` flag to the `forge script` deployment commands.
 
+Alternatively, you can verify the contract after the deployment:
```

Comment:
The `--verify` flag didn't work for me, not sure if I did something wrong. 

I also noticed the contracts were not verified (even the ones deployed from before). I verified manually using the following commands:

Reply:
I added additional comments. 

Also, I saw `--verify` works on the first deployment